base/src/main/java/io/spine/validate/FieldValue.java

```diff
@@ -89,6 +92,19 @@ private FieldValue(List<T> values, FieldContext context, FieldDeclaration declar
         FieldDescriptor fieldDescriptor = context.getTarget();
         FieldDeclaration declaration = new FieldDeclaration(fieldDescriptor);
 
+        FieldValue<T> result = resolveType(context, value, declaration);
+        return result;
+    }
+
+    @SuppressWarnings({
+            "unchecked", /* Since the field value is created by validating builders,
```

Comment:
Please transform this long suppression comment into Javadoc. And then, reference the Javadoc from the warning suppression.